src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -82,9 +82,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _actualFont{ DEFAULT_FONT_FACE, 0, DEFAULT_FONT_WEIGHT, { 0, DEFAULT_FONT_SIZE }, CP_UTF8, false }
     {
         _settings = winrt::make_self<implementation::ControlSettings>(settings, unfocusedAppearance);
-
         _terminal = std::make_shared<::Microsoft::Terminal::Core::Terminal>();
 
+        _setupDispatcherAndCallbacks();

```

Comment:
I feel more comfortable if the `_dispatcher` field is readily useable before calling any other member functions. I think it's a member we might use more often in the future. Also, the `_dispatcher` is used in the callbacks that we hook up below, so there's also that.